src/controllers/graphql/types/reply/replyType.js

```diff
@@ -21,4 +21,8 @@ type Query {
     replyCountById(postId: Int): Int
 }
 
+type Mutation {
+  # creates or edits a reply
+  broadcastReply(postId: Int, body: String): Reply
```

Comment:
It's good practice to do:

```
type Mutation {
   broadcastReply(input: BroadcastReplyInput!): Reply
}

input BroadcastReplyInput {
   postId: Int!
   body: String!
}
```
The syntax might be off so double check but this is the general idea

Reply:
neat.  Will update